simulation/include/traccc/simulation/smearing_writer.hpp

```diff
@@ -74,14 +74,14 @@ struct smearing_writer : detray::actor {
             const auto mom = track.mom();
 
             particle.particle_id = particle_id;
-            particle.vx = pos[0];
-            particle.vy = pos[1];
-            particle.vz = pos[2];
-            particle.vt = track.time();
-            particle.px = mom[0];
-            particle.py = mom[1];
-            particle.pz = mom[2];
-            particle.q = track.charge();
+            particle.vx = static_cast<float>(pos[0]);
```

Comment:
Also here. Or is the "EDM" not using `traccc::scalar` here? :confused: If the "EDM" uses `float`, then of course you're right, and this is how it should go...